sdk/digitaltwins/azure-digitaltwins-core/src/test/java/com/azure/digitaltwins/core/DigitalTwinsTestBase.java

```diff
@@ -21,16 +27,16 @@
         .get("CLIENT_ID", "clientId");
 
     protected static final String DIGITALTWINS_URL = Configuration.getGlobalConfiguration()
-        .get("DIGITALTWINS_URL", "https://playback.api.wus2.digitaltwins.azure.net");
-
-    protected DigitalTwinsClientBuilder getDigitalTwinsClientBuilder() {
-        DigitalTwinsClientBuilder builder = new DigitalTwinsClientBuilder()
-            .endpoint(DIGITALTWINS_URL);
+        .get("DIGITALTWINS_URL", PLAYBACK_ENDPOINT);
 
+    protected DigitalTwinsClientBuilder getDigitalTwinsClientBuilder(HttpClient httpClient, DigitalTwinsServiceVersion serviceVersion) {
+        DigitalTwinsClientBuilder builder = new DigitalTwinsClientBuilder();
         if (interceptorManager.isPlaybackMode()){
             builder.httpClient(interceptorManager.getPlaybackClient());
             // Use fake credentials for playback mode.
             builder.tokenCredential(new FakeCredentials());
+            // Connect to a special host when running tests in playback mode.
```

Comment:
do we really have to set the endpoint on the playback client? it's not going to be using it.
regardless, I think  this comment is incorrect, it doesn't "connect" to anything :)

Reply:
Our builder makes it mandatory that an endpoint is specified, so my hands are tied here

Reply:
The previous implementation just used the playback endpoint if the environment variable for endpoint isn't set, which wasn't quite right. Now we always use the playback endpoint when the interceptorManager is in playback mode